bedrock/mozorg/templates/mozorg/developer/includes/sub-nav.html

```diff
@@ -7,7 +7,26 @@
 {% block sub_nav_class %}technology{% endblock %}
 
 {% block sub_nav_primary_links %}
-<li><a href="{{ url('mozorg.technology') }}" data-link-name="Web Innovations" data-link-type="nav" data-link-position="subnav">{{ _('Web Innovations') }}</a></li>
-<li><a href="{{ url('mozorg.developer') }}" data-link-name="Mozilla Developers" data-link-type="nav" data-link-position="subnav">{{ _('Mozilla Developers') }}</a></li>
-<li><a href="https://developer.mozilla.org" data-link-name="MDN" data-link-type="nav" data-link-position="subnav">{{ _('MDN') }}</a></li>
+<li>
+  {% if 'technology' in request.path %}
```

Comment:
I know we're waiting on some feedback, but flagging that we may not need this logic

Reply:
Having seen that there is a link in the primary nav now, I agree that this is not needed but, yeah, let's wait on feedback in the bug to be certain.